### ISSUE TITLE
Dockerfile broken?

### ISSUE THREAD

**Issue by pgensler** [andrewheiss/tidyverse-rstanarm#1]
When I try to run your dockerfile, I seem to get an error about this line failing:
COPY R/Makevars /root/.R/Makevars

If I comment that out, I get an issue with compiling rstan, which I am unsure about how to resolve:
```
                     ^~~~
make: *** [lang__grammars__expression_grammar_inst.o] Error 1
ERROR: compilation failed for package ‘rstan’
* removing ‘/usr/local/lib/R/site-library/rstan’
```
which I have yet to find a workaround that works properly. Would you know how to fix this so rstan compiles correctly?
EDIT::Sorry, after adding the Makevars file, the image seems to work. Can you explain what exactly the Makevars file does for the container? Not sure I fully understand why it is needed.
Related: https://github.com/stan-dev/rstan/issues/441

**Comment by ignacio82** [comment]
@pgensler were you able to get this to work?

**Comment by pgensler** [comment]
@ignacio82 let me check my build logs, my container is here FYI:
https://hub.docker.com/r/pgensler/sandboxr/

**Comment by pgensler** [comment]
Yes I think it's working, I'm not sure if I have the correct flags for my compiler. I think my Makevars file was not copying over correctly, which was causing the error.
